src/main/java/com/osiris/autoplug/client/tasks/updater/search/CustomCheckURL.java

```diff
@@ -0,0 +1,79 @@
+package com.osiris.autoplug.client.tasks.updater.search;
+
+import com.google.gson.JsonObject;
+import com.osiris.autoplug.client.tasks.updater.plugins.MinecraftPlugin;
+import com.osiris.autoplug.client.tasks.updater.search.SearchResult;
+import com.osiris.autoplug.client.utils.UtilsURL;
+import com.osiris.jlib.json.Json;
+import com.osiris.jlib.logger.AL;
+
+import java.io.File;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CustomUpdateCheck {
+
+    private boolean isInt(String s) {
+        try {
+            Integer.parseInt(s);
+            return true;
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+    public SearchResult doCustomCheck(MinecraftPlugin plugin) {
```

Comment:
What is "FORCE" supposed to do?

Reply:
Allows the user to put "FORCE" as the custom url to force an update without actually checking if there is a newer version.

Reply:
@vengestone-dragoon Okay this could be a separate yaml section 'force-update' which is false by default. Note that usually the check is still needed to get the download url.

Reply:
Ill update that to be a new part of the config

Reply:
Done